65.md

```diff
@@ -6,11 +6,9 @@ Relay List Metadata
 
 `draft` `optional`
 
-Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
+This NIP defines a series of replaceable event kinds for advertising user relay preferences, each with a particular purpose. These events MUST include a list of `r` tags with relay URIs. The `.content` field is not used.
```

Comment:
> These events MUST include a list of `r` tags

I think there was a discussion to deprecate r tags because of indexing. If we introduce new kinds we should go with `relay` instead of `r` imo

Reply:
I'm willing to change it, it's annoying though because it introduces an inconsistent use of tags in a single NIP. It might be useful to have